iterative/utils/logger_test.go

```diff
@@ -16,7 +17,8 @@ func TestState(t *testing.T) {
 		},
 	})
 
-	logger := TpiLogger(d)
+	logger := logrus.WithFields(logrus.Fields{"d": d})
+	logrus.SetFormatter(&TpiFormatter{})
```

Comment:
thins kind of thing just begging for a helper function seeing as it's used a lot?

Reply:
Not worth the hassle: it's used often in “tests” but only once in the code.